src/components/ClientQuote/ClientQuote.tsx

```diff
@@ -0,0 +1,41 @@
+import styles from "./ClientQuote.module.scss";
+import finishLine from "../../assets/images/finishLine.jpg";
+import Image from "next/image";
+
+export function ClientQuote() {
+  return (
+    <main className={styles.main}>
```

Comment:
related to my other comment. <section> is the html element you'd want here wrapping this